server/db/connection.js

```diff
@@ -1,20 +1,26 @@
 import "dotenv/config";
-import mysql from "mysql2/promise";
+import pkg from "pg";
+const { Client } = pkg;
 
 const database = {
   database: process.env.DB_DATABASE,
   user: process.env.DB_USER,
   password: process.env.DB_PASSWORD,
   host: process.env.DB_HOST,
+  port: process.env.DB_PORT,
 };
 
 const testDatabase = {
   database: process.env.DB_TEST_DATABASE,
   user: process.env.DB_TEST_USER,
   password: process.env.DB_TEST_PASSWORD,
   host: process.env.DB_TEST_HOST,
+  port: process.env.DB_TEST_PORT,
```

Comment:
This should be added to `.env.example`

Reply:
Done https://github.com/michaelti/play-up/pull/36/commits/270850f0c9923ae08fed2045ba43d6e30f6124a3